frontend/src/core/MarimoApp.tsx

```diff
@@ -45,7 +46,15 @@ export const MarimoApp: React.FC = () => {
     <ErrorBoundary>
       <TooltipProvider>
         <DayPickerProvider initialProps={{}}>
-          <ModalProvider>{body}</ModalProvider>
+          <ModalProvider>
+            <CssVariables
+              variables={{
+                "--marimo-api-code-editor-font-size": `${userConfig.display.code_editor_font_size}px`,
```

Comment:
creating a pattern to have "public api" css variable tokens. these are prefixed with `--marimo-api`